tests/cypress/integration/4-design-steps/typography-step.cy.js

```diff
@@ -31,8 +23,8 @@ describe( 'Typography Step Test', function () {
 		);
 	} );
 
-	it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
-		CheckIntroPanel( '__design-typography', 'Typography' );
+	it.skip( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
```

Comment:
Why are we skipping this one?
It see that it is working in Colors.

Reply:
Yes, I have to remove it because when we directly go to the page using URL the sidebar isn't present.
<img width="1792" alt="image" src="https://github.com/newfold-labs/wp-module-onboarding/assets/79664985/d664eac8-195e-4666-b844-b5954ea7c576">
have a look at this screenshot.

Reply:
Damn that should be a bug! 😕

Reply:
I'll log a defect for this

Reply:
https://jira.newfold.com/browse/PRESS2-1421